R/plotting/access_change_area.R

```diff
@@ -0,0 +1,61 @@
+library("tidyverse")
+library("config")
+library("here")
+
+# Read files ----
+
+file_path <-
+  here("results", config::get("projected_scenario"), "agents.rds")
+agents <- read_rds(file_path)
+
+# Group agents tables ----
+
+res_vars <- c("nr_tours0", "nr_tours1",
+              "total_access0", "total_access1")
+
+agents <- agents %>%
+  group_by(area) %>%
+  summarise_at(res_vars, sum, na.rm = TRUE)
+
+# Calc differences ----
+
+agents <- agents %>%
+  mutate(
+    projected = total_access1 / nr_tours1,
+    baseline = total_access0 / nr_tours0,
+    util_dif = projected - baseline
+  )
+
+# Plot ----
+max_dif <- 1
+
+agents %>%
+  ggplot(aes(x = area, y = util_dif)) +
+  geom_bar(
+    stat = "identity",
+    position = "dodge",
+    color = "white",
+    fill = hsl_cols("blue"),
+    width = 0.8
+  ) +
+  ylim(-max_dif, max_dif) +
+  theme_fig +
+  geom_abline(slope = 0) +
+  labs(
+    y = "Difference in expected utility (eur) / tour",
+    x = NULL,
+    title = paste0(
+      "Change in average tours' accessibility: ",
+      config::get("projected_name")
+    )
```

Comment:
If there is still budget left, could the plots be in Finnish? Titles, axis, labels, legends... Now they are in English.
This goes for all plots.

Reply:
Yep, we'll need to translate these at some point. Now Helmet model produces agent data with English variable data (age, gender, area). What do you think would be best place and way to translate them? @johpiip @zptro

Reply:
@atte-wsp Is `combine_agent_data.R` collecting all needed data for plotting? Ages and areas are mutated into factors on rows 89-102. I would translate the labels there with [forcats::recode](https://forcats.tidyverse.org/reference/fct_recode.html). That way helmet-model-system would still produce English labels. I think the philosophy is to keep helmet-model-system in English (correct me if I'm wrong, @zptro).

Reply:
I agree with @johpiip!